src/components/AppLayout/Header/components/ProviderDetails/ConnectDetails.tsx

```diff
@@ -1,5 +1,5 @@
 import { withStyles } from '@material-ui/core/styles'
-import * as React from 'react'
+import { ReactElement } from 'react'
```

Comment:
you can actually access it via `React.ReactElement` without importing, `React` namespace is available globally for types